### PR TITLE
Twitter sample improvements

### PR DESCRIPTION
**Pull request by frsela**
Added support to the new API and now is able to reconnect and recover pending messages.
Added support to multiple streams & localized ones

### CODE REVIEW

File: examples/twitter/twitter_client/js/push.js [hunk 1]
```diff
@@ -159,21 +174,40 @@ var Push = {
...
+              watoken: "publicTwitterStream",
+              pbkbase64: utf8_to_b64(this.pbk)
+            },
+            messageType: "registerWA"
+          });
+        } else {
+          this.logMessage('[MSG registerUA] Recovering pending messages');
+          for(var i in msg.messages) {
+            this.notificationMessage(msg.messages[i].payload.message);
+            this.sendWS({
```
Comment (frsela) [inline examples/twitter/twitter_client/js/push.js:200]: @willyaranda When recovering messages from the registerUA response, we need to check the payload. This is different with the notification message. I suggest that the array list only contains payloads

Comment (willyaranda) [inline examples/twitter/twitter_client/js/push.js:200]: Implies more computation at server side, but we could do that, yes

Comment (frsela) [inline examples/twitter/twitter_client/js/push.js:200]: I think the rest of the info is a server internal data. Other implementation can have different way of storage so I see strange to send all raw data :s

What about store only payloads on nodes collection?

Comment (willyaranda) [inline examples/twitter/twitter_client/js/push.js:200]: Inside payloads we also add the messageId and the URL, so we are modifying it. We can send that payload and not the whole object.

Comment (willyaranda) [inline examples/twitter/twitter_client/js/push.js:200]: Once my PR lands, check for `msg.messages[i].message`

File: examples/twitter/twitter_client/js/push.js [hunk 2]
```diff
@@ -159,21 +175,40 @@ var Push = {
...
+              messageType: "ack",
+              messageId: msg.messages[i].payload.messageId
```
Comment (willyaranda) [inline examples/twitter/twitter_client/js/push.js]: `msg.messages[i].messageId` here then

**Pull request merged**